cpp/data/system_updates/127.sh

```diff
@@ -0,0 +1,26 @@
+# Insert is_publisher provided also to the HTML cache
+#!/bin/bash
```

Comment:
I thought that `#!/bin/bash` always needs to be in the first line of a .sh file or else it won't have any effect and will just be treated as a regular comment